plycpp/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.8)
+project(plycpp)
+
+add_compile_options(-Wall -Wextra -Wpedantic)
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+
+get_filename_component(MODELS_DIRECTORY
+  "${CMAKE_CURRENT_SOURCE_DIR}/models"
+  ABSOLUTE)
+
+include_directories(
+  PUBLIC
+  $<INSTALL_INTERFACE:include>
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+)
+
+add_library(plycpp SHARED src/plycpp.cpp)
+
+install(
+  TARGETS plycpp
+  EXPORT plycpp)
+
+target_include_directories(${PROJECT_NAME}
```

Comment:
target_include_directories(${PROJECT_NAME}
  PUBLIC
  $ <INSTALL_INTERFACE:include>
  $ <BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
)
(remove white space after $, I had to add it to avoid weird formatting here :) )